database/src/tools/IAPP_Migrator.ts

```diff
@@ -3,9 +3,42 @@ import { AxiosRequestConfig } from 'axios';
 import csv from 'csvtojson';
 import axios from 'axios';
 import qs from 'qs';
+import moment from 'moment';
 
 const meow = require('meow');
 
+const formatDateToISO = (d) => {
+  if (!d) return undefined;
```

Comment:
should just be able to say
```suggestion
  if (!d) return;
```